config/config.yml

```diff
@@ -0,0 +1,124 @@
+# Network Configuration
```

Comment:
Maybe we could call this file `default-config.yml`?

Reply:
https://github.com/onflow/flow-go/pull/4340/commits/143d8ecefb5db4670f347b832413a8291e0e333e